app/src/main/java/com/plan2meet/ui/RetrofitClientInstance.kt

```diff
@@ -6,15 +6,16 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = ""
+    //never start a variable with an uppercase.
```

Comment:
0) The original implementation is correct.  All upper case, separated by underscores, is proper convention for constants.  This appears to be a constant, though I'd add "const" modifier to make that more clear.  
1) Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.